ui/app/templates/components/dashboard/quick-actions-card.hbs

```diff
@@ -1,74 +1,87 @@
 <Hds::Card::Container @hasBorder={{true}} class="has-padding-l" data-test-card="quick-actions">
   <h3 class="title is-4">Quick actions</h3>
-  <div class="has-top-margin-m has-bottom-margin-m">
-    <h4 class="title is-marginless is-6">Secrets engines</h4>
-    <p class="is-size-8 has-top-margin-xxs has-bottom-margin-s has-text-grey">Supported engines include databases, KV version
-      2, and PKI.</p>
-    <SearchSelect
-      @id="secrets-engines-select"
-      @options={{this.mountOptions}}
-      @selectLimit="1"
-      @disallowNewItems={{true}}
-      @fallbackComponent="input-search"
-      @onChange={{this.handleSearchEngineSelect}}
-      @placeholder="Type to select a mount"
-      @displayInherit={{true}}
-      @shouldRenderName={{true}}
-      @passObject={{true}}
-      @objectKeys={{array "type" "version"}}
-      class="is-marginless"
-      data-test-secrets-engines-select
-    />
-  </div>
-
-  {{#if this.selectedEngine}}
-    <h4 class="title is-6">Action</h4>
-    <Select
-      @name="action-select"
-      @options={{this.actionOptions}}
-      @isFullwidth={{true}}
-      @selectedValue={{this.selectedAction}}
-      @onChange={{this.setSelectedAction}}
-      @noDefault={{true}}
-    />
-
-    {{#if this.searchSelectParams.model}}
-      <h4 class="title is-6" data-test-search-select-params-title>{{this.searchSelectParams.title}}</h4>
-
+  {{#if this.filteredSecretEngines.length}}
```

Comment:
It looks like `filteredSecretEngines` could be undefined or an empty array? I think templates fail more gracefully than javascript methods (this would error if `filteredSecretEngines` was undefined and we tried calling `.length` on it.)

Since an empty array is falsy, I think we could also just check for `{{#if this.filteredSecretEngines}}`